### PR TITLE
play next

### PR DESCRIPTION
**Pull request by halevibarak**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -4,8 +4,46 @@ apply plugin: 'kotlin-android'
...
+    }
+
+    buildTypes {
+        debug {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
+
+        release {
+            minifyEnabled false
```
Comment (PRuedaApplicaster) [inline android/build.gradle:27]: set to true for release please

File: android/src/main/java/com/applicaster/plugin/televisionacademyplayer/TAPlayerActivity.kt [hunk 2]
```diff
@@ -104,39 +159,61 @@ class TAPlayerActivity : AppCompatActivity() {
...
+            currentPlaylistItem += 1
+        }
+        playable!!.setContentVideoUrl(this.playlistItems[currentPlaylistItem])
+        val sourceConfiguration = SourceConfiguration()
+        sourceConfiguration.addSourceItem(playable!!.contentVideoURL)
+        sourceConfiguration.startOffset = currentProgress
+        bitmovinPlayer?.load(sourceConfiguration)
+        EventListenerInteractor.addListeners(bitmovinPlayer, playable?.playableId
+                ?: "", contentGroup)
+        Log.e(TAG, "play now: " + currentPlaylistItem + " || current:" + currentProgress + " from total items:" + playlistItems.size)
```
Comment (PRuedaApplicaster) [inline android/src/main/java/com/applicaster/plugin/televisionacademyplayer/TAPlayerActivity.kt:217]: Please remove this log or at least display as debug info, not as an error.

File: android/src/main/java/com/applicaster/plugin/televisionacademyplayer/network/NetworkAPI.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import retrofit2.Call
+import retrofit2.http.*
+
+interface NetworkAPI {
+
+    @GET("playback/{content_uid}?manifest_type=HLS")
+    @Headers("Accept: application/json")
+    fun getPlaybackURL(@Path("content_uid") forContentId : String, @Header("Authorization") token : String) : Call<PlaybackURLResponse>
+
+    @GET("fetchData?type=submissions&screen=videos&env=prod&isTVApp=false")
```
Comment (PRuedaApplicaster) [inline android/src/main/java/com/applicaster/plugin/televisionacademyplayer/network/NetworkAPI.kt:13]: Is it possible to expose this in the plugin configuration?

File: android/build.gradle (hunk 1 above)
Comment (halevibarak) [reply android/build.gradle:27]: done

File: android/src/main/java/com/applicaster/plugin/televisionacademyplayer/network/NetworkAPI.kt (hunk 3 above)
Comment (halevibarak) [reply android/src/main/java/com/applicaster/plugin/televisionacademyplayer/network/NetworkAPI.kt]: done by migrate network to OKHttp

**Pull request merged**